source/read-write-concern/tests/connection-string/write-concern.json

```diff
@@ -0,0 +1,118 @@
+{
+  "tests": [
+    {
+      "description": "Default",
+      "uri": "mongodb://localhost/",
+      "valid": true,
+      "warning": false,
+      "writeConcern": {}
+    },
+    {
+      "description": "w as a valid number",
+      "uri": "mongodb://localhost/?w=1",
+      "valid": true,
+      "warning": false,
+      "writeConcern": {
+        "w": 1
+      }
+    },
+    {
+      "description": "w as an invalid number",
+      "uri": "mongodb://localhost/?w=-2",
+      "valid": false,
+      "warning": null
+    },
+    {
+      "description": "w as a string",
```

Comment:
I'd update this to say "w as majority" and add at least one test for w=random-gle-tag